test/e2e/preflightCliTest.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+)
+
+func preflightHelp() string {
+	cmd := exec.Command("preflight")
+	output, err := cmd.Output()
+
+	if err != nil {
+		fmt.Println("error:", err)
+	} else {
+		//#fmt.Printf("output: %s \n\n\n%s", string(output), string(want))
+	}
+	return string(output)
+}
```

Comment:
IMO, for this you can take a look how we run shell commands and return an output that contains both stdout and stderr. You can then run some checks against stdout/stderr/error.

https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/da9e057934e43eefc765b0be3505b2b07630d23b/certification/internal/engine/podman.go#L48